packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -148,7 +148,9 @@ export const createEditorView = (
   const blocksArray = Object.values(blocks);
 
   const paragraphBlock = blocksArray.find(
-    (block) => block.meta.name === "@hashintel/block-paragraph",
+    (block) =>
+      block.meta.componentId ===
+      "https://blockprotocol.org/blocks/@hash/paragraph",
```

Comment:
Should this not be `https://blockprotocol.org/@hash/blocks/paragraph`?

Reply:
Also to flag, this actual resolving page above ^ contains a link to `https://blockprotocol.org/@hash/paragraph` in the description, which 404s now that blocks are namespaced separately from types.

Reply:
> Also to flag, this actual resolving page above ^ contains a link to https://blockprotocol.org/@hash/paragraph in the description, which 404s now that blocks are namespaced separately from types.

I've removed these (worth noting that these pages aren't actually discoverable at the moment, since the text blocks are hidden from the hub)